lib/dbadaptor.py

```diff
@@ -150,28 +156,23 @@ def update(self, conn, table_name, values, where, ph):
         if where is not None:
             sql = sql + 'WHERE ' + where
         sql_cmd = sql % (table_name, sets)
-        c.execute(sql_cmd, vals)
+        with closing(conn.cursor()) as c:
+            c.execute(sql_cmd, vals)
         conn.commit()
 
-    def delete(self, conn, table_name, where):
-        '''Update data in a table
-        @conn A connection of database
+    def remove(self, conn, table_name, where):
+        '''Remove rows based on specified conditions
```

Comment:
Is there any reason to change the name of this method?  `delete` is a valid command in sql language.

Reply:
I changed it to remove because the pysixdb method is called remove, so for consistency when I did the refactoring. But it does not make sense now.